grakn-core/src/main/java/ai/grakn/util/GraqlSyntax.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016-2018 Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/agpl.txt>.
+ */
+
+package ai.grakn.util;
+
+/**
+ * Graql syntax keywords
+ *
+ * @author Haikal Pribadi
+ */
+public class GraqlSyntax {
```

Comment:
what's this for? how do we ensure it's in sync with the antlr grammar file?

Reply:
just to document my answer to @kasper-piskorski in person:

We need this class so we can replace all usage of hard-coded strings of Graql's syntax across our code base. It will be much less error-prone and much easier to manage when we change the syntax. 

It may appear that we have GraqlParser (produced by ANTLR) which contains Graql Syntax' constants, but these constants are integers that are generated by ANTLR, which may change after compiling updates from the grammar file.

@lolski @duckofyork